<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageBox</class>
 <widget class="QDialog" name="MessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>100</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/Icons.qrc">
    <normaloff>:/DSV2.ico</normaloff>:/DSV2.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{background-color: rgb(50, 50, 51);
color: rgb(230, 230, 230);}
QLabel{color:rgb(230, 230, 230);}
</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lb_icon">
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_text">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	color:rgb(230, 230, 230);
	background-color:rgb(70, 70, 71);
	border:none;
	min-width:80;
	min-height:24;
}
QPushButton:hover
{
	background-color:rgb(0, 85, 127);
}
QPushButton:pressed
{
	background-color:rgb(0, 85, 127);
}
</string>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::No|QDialogButtonBox::StandardButton::Yes</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../res/Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>MessageBox</receiver>
   <slot>button_clicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>button_clicked(QAbstractButton*)</slot>
 </slots>
</ui>
